src/components/views/Jobs.js

```diff
@@ -145,24 +147,50 @@ export default class Jobs extends React.Component {
     );
   }
 
+  renderNewJobMenu = ({ onToggle }) => (
+    <DropdownMenu
+      data-role="menu"
+      aria-label={<FormattedMessage id="ui-local-kb-admin.job.availableJobTypes" />}
+      onToggle={onToggle}
+    >
+      <FormattedMessage id="ui-local-kb-admin.job.newJob">
+        {ariaLabel => (
+          <>
+            <Button
+              aria-label={ariaLabel}
+              buttonStyle="dropdownItem"
+              id="clickable-new-JSON-job"
+              marginBottom0
+              to={`/local-kb-admin/create/JSON${this.props.searchString}`}
+            >
+              <FormattedMessage id="ui-local-kb-admin.job.newJSONJob" />
+            </Button>
+            <Button
+              aria-label={ariaLabel}
+              buttonStyle="dropdownItem"
+              id="clickable-new-KBART-job"
+              marginBottom0
+              to={`/local-kb-admin/create/KBART${this.props.searchString}`}
+            >
+              <FormattedMessage id="ui-local-kb-admin.job.newKBARTJob" />
+            </Button>
+          </>
+        )}
+      </FormattedMessage>
+    </DropdownMenu>
+  );
+
   renderResultsLastMenu() {
+    const buttonProps = {
```

Comment:
No need to have a separate variable here is there? Just inline it in at the usage.

Reply:
Inlined